src/Sarif/Core/SarifLog.cs

```diff
@@ -44,5 +44,31 @@ public static SarifLog Load(string sarifFilePath)
                 return serializer.Deserialize<SarifLog>(jtr);
             }
         }
+
+        /// <summary>
+        ///  Write a SARIF log to disk as a file.
+        /// </summary>
+        /// <param name="sarifFilePath">File Path to Sarif file to write to</param>
+        public void Save(string sarifFilePath)
+        {
+            using (StreamWriter writer = File.CreateText(sarifFilePath))
```

Comment:
>File.CreateText [](start = 41, length = 15)

We may require all IO in this file to go through our test IFileSystem interface in future, just a heads up, no note for now.

Reply:
Ok. I will definitely try to add both Stream and file overloads at least; Stream is the most broadly useful and file path is an overload that code consumers want to see there so they don't have to think. Happy to add others to suit design rules if needed.